src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -77,14 +78,14 @@ Instance configuration is generated by Che and is updated by our internal config
 ----
 
 [id="jdbc-configuration"]
-== JDBC Configuration
+== JDBC configuration
 
-Eclipse Che uses http://www.h2database.com/html/main.html[H2] for single-user builds and PostgreSQL database in a multi-user flavor.
+Eclipse Che uses http://www.h2database.com/html/main.html[H2] for single-user builds and PostgreSQL database in the multi-user flavor.
 
-Depending on the used database, JDBC Connection pool will be initialized with respective default values. These values can be overridden through che.env.
+Depending on the used database, JDBC Connection pool will be initialized with respective default values. These values can be overridden through the che.env file.
```

Comment:
"the JDBC Connection pool"

Also, is there a reason "Connection" is capitalized here? I'm not familiar with the subject enough to say it shouldn't be, but it seems odd at first glance.

Reply:
It should have been JDBC connection pool". Thanks for pointing that out!